eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Gov.hs

```diff
@@ -133,15 +136,15 @@ data ConwayGovPredFailure era
   | -- | Some governance actions are not allowed to be voted on by certain types of
     -- Voters. This failure lists all governance action ids with their respective voters
     -- that are not allowed to vote on those governance actions.
-    DisallowedVoters !(Map.Map (GovActionId (EraCrypto era)) (Voter (EraCrypto era)))
+    DisallowedVoters !(NonEmpty (GovActionId (EraCrypto era), Voter (EraCrypto era)))
```

Comment:
I now this is not important, but I think voter should come first, since voter is the offender.
```suggestion
    DisallowedVoters !(NonEmpty (Voter (EraCrypto era), GovActionId (EraCrypto era)))
```